test/interface/test_graceful_shutdown.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
+# SPDX-License-Identifier: MIT
+import sys
+import subprocess
+import time
+import signal
+
+
+def test_quiet_exiting():
+    proc = subprocess.Popen(
+        [sys.executable, 'aibolit'],
```

Comment:
Since the error emitted by `rultor` was related to `runpy`, I assume that maybe running this explicitly as a module may help:
```suggestion
        [sys.executable, '-m', 'aibolit'],
```